app/views/activities/index.html.haml

```diff
@@ -0,0 +1,25 @@
+.container
+  .row
+    .col-md-8
+      %h3.align-center.blue View Previous Logs
+      %form.form-inline
+        #date-range.input-group
+          %input.form-control.date-picker{:type => "text", :name => "start"} 
```

Comment:
Yes, this is where the data is coming from. So, we need to change the name attribute to "start_date".